Sources/ArcGISToolkit/Components/Offline/PreplannedListItemView.swift

```diff
@@ -55,6 +74,20 @@ struct PreplannedListItemView: View {
             .font(.body)
     }
     
+    @ViewBuilder private var deleteButton: some View {
+        if model.status.allowsRemoval {
+            Button {
+                Task {
+                    await model.removeDownloadedPreplannedMapArea()
+                    onMapDeletion()
+                }
+            } label: {
+                Label("Remove Area", systemImage: "trash")
```

Comment:
```suggestion
                Text("Delete")
```
`Delete` should be used here to match the "Delete Map Area" button in the metadata view. Is the icon necessary?

<img src="https://github.com/user-attachments/assets/bd037a12-5b8d-487c-806f-55bf2e813cbb" width="300">